howso/react_aggregate.amlg

```diff
@@ -2,153 +2,152 @@
+			;               predict the feature.  When true, computes, caches, and returns the residuals for all features. Uses full computation.
+			;               false removes cached values. When "prediction_stats" in the "details" parameter is true, it will also compute and cache the feature residuals.
+			;		feature_residuals_robust: optional, none/true/false. For each context_feature, computes, caches, and returns the same stats as residuals but using the robust
+			;               (power set/permutations) set of all other context_features to predict the feature. false removes cached values. Uses robust computation.
+			;		feature_contributions_full: optional, none/true/false. For each context_feature, use the full set of all other context_features to compute the
+			;				mean absolute delta between prediction of action_feature with and without the context_feature in the model. false removes cached values. Uses full computation.
+			;		feature_contributions_robust: optional, none/true/false. For each context_feature, use the robust (power set/permutation) set of all other context_features
+			;				to compute the mean absolute delta between prediction of action_feature with and without the context_feature in the model. Uses robust computation.
+			;				false removes cached values. Uses robust computation.
+			;		feature_mda_full: optional, none/true/false. if true will compute Mean Decrease in Accuracy (feature_mda) for each context feature at predicting mda_action_features.
+			;				Drop each feature and use the full set of remaining context features for each prediction. false removes cached values. Uses full computation.
+			;		feature_mda_permutation_full: optional, none/true/false. Compute feature_mda_full by scrambling each feature and using the full set of remaining context features
+			;				for each prediction.  false removes cached values. Uses full computation.
+			;		feature_mda_robust: optional, none/true/false. Compute feature_mda by dropping each feature and using the robust (power set/permutations) set of
+			;				remaining context features for each prediction. false removes cached values. Uses robust computation.
+			;		feature_mda_permutation_robust: optional, none/true/false. Compute feature_mda by scrambling each feature and using the robust (power set/permutations)
+			;				set of remaining context features for each prediction.  false removes cached values. Uses robust computation.
+			;		selected_prediction_stats": list of strings, optional.  Allowed values are:
+			;						"mda" : mean decrease in accuracy when each feature is dropped from the model, applies to all features.
+			;						"contribution": feature contribution to predicted value when each feature is dropped from the model, applies to all features.
+			;						"mda_permutation": mean decrease in accuracy that used scrambling of feature values instead of dropping each feature, applies to all features.
```

Comment:
These actually got removed from this list in a recent PR. Probably was hard to catch in the merge conflict, remove them here please.

Reply:
Checked against main to be sure the enumerated parameters matched per your comment. Addressed in: [d54a657](https://github.com/howsoai/howso-engine/pull/239/commits/d54a6575c30a5c75415b2cc52e96d51cfb0406c6)